### PR TITLE
- Fixed Typos , Broken link to heroku documentation , added a more clear instruction

### PR DESCRIPTION
**Pull request by sanchittechnogeek**
- Fixed typos (navigae to navigate , mdify to modify, etc.)
- Fixed broken link to heroku documentation
- Added a more clear instruction (step no. 8) in all installation instruction files
- Added a new device in tested devices list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,11 +114,11 @@ To install system with Docker please refer to the [Docker installation readme](/
 
 #### How do I deploy Web App Generator with Heroku
 
-Please read how to deploy to [Heroku here](/docs/HEROKU.md)
+Please read how to deploy to [Heroku here](/docs/INSTALLATION_HEROKU.md)
 
 ### Accessing the App Generator
 
-An app generator can be accessed in two ways, via an App Generator Web-Page, that runs on an the APACHE server or via POST API. The POST API takes input in form of a JSON and then creates and emails you the app. To access it via a POST API in your app installation you need to go to http://your-URL/api/api.php. The POST API takes input in form of a JSON and then creates and emails you the app.
+An app generator can be accessed in two ways, via an App Generator Web-Page that runs on an the APACHE server or via POST API. The POST API takes input in the form of JSON and then creates and send you the app in the form of an .apk file via an email. To access it via a POST API in your app installation you need to go to http://your-URL/api/api.php. The POST API takes input in the form of JSON and then creates and send you the app via an email.
```
Comment (mananwason) [inline README.md]: *sends, 
Just .apk, no file

Comment (sanchittechnogeek) [inline README.md]: @mananwason fixed!

**Comment by sanchittechnogeek** [comment]
@mananwason fixed , please merge it!

**Comment by niccokunzmann** [comment]
This is a huge change with the name "fixed typos" - which could be more descriptive.
I think it is valuable, since it adds many descriptions and has a consistent formatting.

**Pull request merged**